routing/probability_estimator_test.go

```diff
@@ -0,0 +1,91 @@
+package routing
+
+import (
+	"math"
+	"testing"
+	"time"
+
+	"github.com/lightningnetwork/lnd/routing/route"
+)
+
+// Create a set of test results.
+var resultTime = time.Unix(1674169200, 0) // 20.01.2023
+var now = time.Unix(1674190800, 0)        // 6 hours later
+var results = NodeResults{
+	route.Vertex{byte(0)}: TimedPairResult{
+		FailAmt:     200_000_000,
+		FailTime:    resultTime,
+		SuccessAmt:  100_000_000,
+		SuccessTime: resultTime,
+	},
+	route.Vertex{byte(1)}: TimedPairResult{
+		FailAmt:     200_000_000,
+		FailTime:    resultTime,
+		SuccessAmt:  100_000_000,
+		SuccessTime: resultTime,
+	},
+	route.Vertex{byte(2)}: TimedPairResult{
+		FailAmt:     200_000_000,
+		FailTime:    resultTime,
+		SuccessAmt:  100_000_000,
+		SuccessTime: resultTime,
+	},
+	route.Vertex{byte(3)}: TimedPairResult{
+		FailAmt:     200_000_000,
+		FailTime:    resultTime,
+		SuccessAmt:  100_000_000,
+		SuccessTime: resultTime,
+	},
+	route.Vertex{byte(4)}: TimedPairResult{
+		FailAmt:     200_000_000,
+		FailTime:    resultTime,
+		SuccessAmt:  100_000_000,
+		SuccessTime: resultTime,
+	},
+}
+
+// BenchmarkBimodalPairProbability benchmarks the probability calculation.
+func BenchmarkBimodalPairProbability(b *testing.B) {
+	estimator := BimodalEstimator{
+		BimodalConfig: BimodalConfig{
+			BimodalScaleMsat:  scale,
+			BimodalNodeWeight: 0.2,
+			BimodalHalfLife:   48 * time.Hour,
+		},
+	}
+
+	b.ResetTimer()
+
+	toNode := route.Vertex{byte(0)}
+	for i := 0; i < b.N; i++ {
+		estimator.PairProbability(now, results, toNode,
+			150_000_000, 300_000)
+	}
+}
+
+// BenchmarkAprioriPairProbability benchmarks the probability calculation.
+func BenchmarkAprioriPairProbability(b *testing.B) {
```

Comment:
Thx for adding this set of benchmarks! 

I think we can leave them in the tree as well, so this commit doesn't necessarily need to be temp.